app/views/course/assessment/submissions/_guided.html.slim

```diff
@@ -0,0 +1,3 @@
+div.btn-group
```

Comment:
Not really, just a for loop and all the questions are loaded.  Displaying all questions will be useful when we allow to skip questions.